scalafix-cli/src/main/scala/scalafix/internal/v1/Args.scala

```diff
@@ -81,6 +81,11 @@ case class Args(
         "configured in .scalafix.conf or via --rules"
     )
     syntactic: Boolean = false,
+    @Description(
+      "Run only rules defined in onCompile section in .scalafix.conf," +
+        "or rules explicitly passed via --rules"
```

Comment:
Maybe this is a good place to mention the fact that the section only overlays the default stuff?
```suggestion
      "Overlay the default rules & rule settings in .scalafix.conf with the `onCompile` section"
```